wordpress-mobile/WordPress-iOS
# Fixes a styling issue in the username selection screen during signup.

Related to https://github.com/wordpress-mobile/WordPress-iOS/issues/16380

## Associated PRs:

WPAuthenticator: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/591

## To test:

Run this test in both light and dark modes.

1. Run the app
2. Signup with a new user
3. In the signup epilogue screen, tap on the username to pick a different one.
4. Make sure the "Back" and "Help" buttons are visible in the navigation bar (and blue), and that the navigation bar title is also visible (the colors depends on the mode).

## Regression Notes

1. Potential unintended areas of impact

None that I can think of, since this only makes sure the navigation bar style is the correct one under very specific circumstances.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

No action taken.

3. What automated tests I added (or what prevented me from doing so)

None since this is a styling issue.

PR submission checklist:
